### PR TITLE
fixing overriding save() method in models

### PR DESCRIPTION
**Pull request by MAM-SYS**
i faced an issue about overriding some of ***save()*** method in some models like [Industry](https://github.com/jobguywork/backend/blob/ac7dfd3601f9a5e3814c56e8b44ca888cbf22385/company/models.py#L9) or [Company](https://github.com/jobguywork/backend/blob/ac7dfd3601f9a5e3814c56e8b44ca888cbf22385/company/models.py#L74)
i changed parameters to *args and **kwargs to solve this problem
this PR is for #48
hope it could help

### CODE REVIEW

File: authnz/models.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def save(self, *args, **kwargs):
             nick_name = md5(str(time()).encode())
             nick_name.update(self.user.username.encode())
             self.nick_name = nick_name.hexdigest()
-        super().save(*args, **kwargs)
+        super(Profile, self).save(*args, **kwargs)
```
Comment (sbabashahi) [inline authnz/models.py]: redundant Profile and self

File: company/models.py [hunk 2]
```diff
@@ -25,8 +25,8 @@ def company_count(self):
     def __str__(self):
         return self.name
 
-    def save(self, force_insert=False, force_update=False):
-        super(Industry, self).save(force_insert, force_update)
+    def save(self, *args, **kwargs):
+        super(Industry, self).save(*args, **kwargs)
```
Comment (sbabashahi) [inline company/models.py]: Plz remove redundant Industry and self

Comment (sbabashahi) [reply company/models.py]: and do it for all other changed

Comment (MAM-SYS) [reply company/models.py]: excuse me for that
now is it ok?

Comment (sbabashahi) [reply company/models.py]: one last requirement :) please make all of your changes in one commit

Comment (MAM-SYS) [reply company/models.py]: sorry if i did it wrong at first
and thanks again for helping me to do this in a right way
is it ok now ?

Comment (sbabashahi) [reply company/models.py]: thanks for your effort :)

**Review by yazdanv — APPROVED** [review]

**Pull request merged**